.github/workflows/eamxx_default_files.yml

```diff
@@ -0,0 +1,62 @@
+name: inputdata
```

Comment:
what does this do?

Reply:
@mahf708

Reply:
I think it checks that certain inputs are downloaded.

Reply:
This should be harmless for E3SM, right?